File: backend/src/main/kotlin/io/paulocosta/twitbooks/goodreads/Work.java
package io.paulocosta.twitbooks.goodreads;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "work", strict = false)
public class Work {

    public Work() {}

    @Element(name = "best_book")
    private GoodreadsBook bestGoodreadsBook;

    public GoodreadsBook getBestGoodreadsBook() {
        return bestGoodreadsBook;
    }

    public void setBestGoodreadsBook(GoodreadsBook bestGoodreadsBook) {
        this.bestGoodreadsBook = bestGoodreadsBook;
    }
}

File: backend/src/main/kotlin/io/paulocosta/twitbooks/books/provider/goodreads/Work.kt
package io.paulocosta.twitbooks.books.provider.goodreads;

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "work", strict = false)
data class Work @JvmOverloads constructor(
        @field:Element(name = "best_book")
        @param:Element(name = "best_book")
        var bestGoodreadsBook: GoodreadsBook? = null
)
